collective/pwexpiry/scripts/notify_and_expire.py

```diff
@@ -55,7 +55,7 @@ def notify_and_expire():
         notifications_to_use = registry['collective.pwexpiry.notification_actions']
     current_time = portal.ZopeTime()
     local_tz = current_time.timezone()
-    for user_id in portal.acl_users.source_users.getUserIds():
+    for user_id in portal.acl_users.getUserIds():
```

Comment:
don't know enough about acl_users.
can you explain why this change is necessary and if it will work on installations w/o membrane too.